.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+* What does this PR do?
+
+* Are there points in the code the reviewer needs to double check?
+
+* Why was this PR needed?
+
+* Screenshots (if relevant)
+
+* Does this PR meet the acceptance criteria?
+
```

Comment:
now that we've got (or are about to have) a style guide we could add:

- [ ] Code syntax follows the style guide